chaco/polar_mapper.py

```diff
@@ -29,8 +29,11 @@ class PolarMapper(AbstractMapper):
     # Private traits
     # ------------------------------------------------------------------------
 
-    _scale = Float(1.0)  # number of screen space units per data space unit
+    # Number of screen space units per data space unit.
```

Comment:
```suggestion
    #: Number of screen space units per data space unit.
```

Reply:
This are privately named traits, but I agree they do seem useful to have in the api docs. I will go ahead with the # to #: changes

Reply:
Actually, I'm having second thoughts.  I don't think users should need to ever touch these traits correct?